src/systems/physics/Physics.cc

```diff
@@ -129,6 +132,7 @@ class ignition::gazebo::systems::PhysicsPrivate
           ignition::physics::GetBasicJointState,
           ignition::physics::SetBasicJointState,
           ignition::physics::SetJointVelocityCommandFeature,
+          ignition::physics::GetModelBoundingBox,
```

Comment:
I started work to make this feature optional on the `ign-gazebo3` branch here:

https://github.com/ignitionrobotics/ign-gazebo/blob/2259f027cd3a244f2c841b62c968088eace1b338/src/systems/physics/Physics.cc#L370-L387

What do you think of doing this from this PR? It will reduce my PR later. You'd need to forward-port just the [entityCast](https://github.com/ignitionrobotics/ign-gazebo/pull/62/files#diff-2b54a7efa7163b4546a1568fdd51cbbb) function from #58 .

If you need this urgently, the optional feature can wait.

Reply:
ok d18df79 should do it. I notice the one of the `entityCast` args changed between blueprint and your branch. Not sure which one I should use so I picked the blueprint version but should be easy to change.

Reply:
Thanks! Yeah the Blueprint function is the correct one.